app/[[...path]]/page.tsx

```diff
@@ -132,6 +132,17 @@ type MetadataProps = {
   };
 };
 
+// Helper function to clean up canonical tags missing leading or trailing slash
+function checkCanonicalTagFormat(tag: string) {
```

Comment:
super nitpicky: `check...` makes me think this is going to return a boolean rather than a transformed value. Maybe `formatCanonicalTag()`? 100% a judgement call and not blocking this PR whatsoever

Reply:
Thanks @mjq! I went ahead and took your suggestion as well as moving the formatting function call to make things a little more readable.